lib/gooddata/bricks/help_brick.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'brick'
+
+module GoodData
+  module Bricks
+    # Simple brick used for printing help message anout bricks
+    class HelpBrick < GoodData::Bricks::Brick
+      def version
+        '0.0.1'
+      end
+
+      # HelpWorld brick entry-point
+      def call(params)
```

Comment:
info: Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction) (reek)